mesonbuild/mintro.py

```diff
@@ -226,8 +229,13 @@ def run(args):
         testdata = pickle.load(f)
     with open(benchmarkfile, 'rb') as f:
         benchmarkdata = pickle.load(f)
-    with open(installfile, 'rb') as f:
-        installdata = pickle.load(f)
+    # Install data is only available with the Ninja backend
```

Comment:
It really should be available for all backends. Should at least change this comment to state that this is due to the fact that other backend install functionality has not been written yet.